### ISSUE TITLE
Resume an episode from where it got left.

### ISSUE THREAD

**Issue by carvan** [maykar/plex_assistant#160]
Hi there,
Is this something missing from the integration or simply it is not working? I really would like to be able to tell the Google home device to play the series I'm currently watching and be able to resume the playing from where it was left last time I watched it.

Thanks everyone

**Comment by pimp1310** [comment]
add a "resume": 1

**Comment by carvan** [comment]
> add a "resume": 1

That is interesting! Thanks! The thing now is that I really don't know where I should add this.. Currently my setup is this:
- I use IFTTT 
- I configured a webhook command that does trigger the simple play command like this: { "action": "call_service", "service": "plex_assistant.command", "command": "{{TextField}}" }
- When I say "Hey Google, play {{this}} on the media server, it does launch the episode from the beginning
Other than these, I didn't configure anything else. So my question would be, where should I add this setting ?

**Comment by pimp1310** [comment]
yeah sorry i was wrong i trested it in Homeassistant with the wrong command.

it must be implanted in the integration.
is the author still active here or in a forum?

**Comment by carvan** [comment]
@maykar what do you think about this ?
